### ISSUE TITLE
NTR, pulmo: Nasal ulcer

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6478]
**Preferred term label:**
Nasal ulcer

**Synonyms**

**Definition (free text, please give PubMed ID)**
An open sore of the nasal mucosa.
Comment: Nasal ulcers may occur due exogenous factors, such as trauma, pressure or inhalation of metals dust or aerosol. Nasal ulcers may also be a sign of an underlying disease, for example malignancies, infections (tuberculosis, leprosy, leishmaniasis), vasculitis (Wegener’s granulomatosis, Churg-Strauss syndrome) or systemic lupus erythematosus.

•	PMID: 26657735
•	PMID: 11314863 
•	PMID: 25441476 

**Parent term (use hpo.jax.org/app)**
Functional respiratory abnormality HP:0002795
Abnormality of the nasal mucosa HP:0000433

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu: @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
I will add this but I think it is not good to classify it as "functional" (everything is functional in one way or another, but this is a feature that is primarily morphological).

**Comment by pnrobinson** [comment]
added
